core/src/components/progress-bar/test/shape/progress-bar.e2e.ts

```diff
@@ -0,0 +1,56 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+configs({ modes: ['ionic-md'] }).forEach(({ title, screenshot, config }) => {
```

Comment:
Since functionality doesn't change based on direction, we should only test LTR.

```suggestion
configs({ directions: ['ltr'], modes: ['ionic-md'] }).forEach(({ title, screenshot, config }) => {
```